components/RecipeList.js

```diff
@@ -0,0 +1,52 @@
+import data from "@/db/data";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function RecipeList() {
+  if (!data) {
+    return <p>Data not found</p>;
+  }
+
+  return (
+    <RecipeStyledUl>
+      {data.map((recipe) => (
+        <li key={recipe.name}>
+          <StyledFigure>
+            <Image
+              src={recipe.imageURL}
+              alt={recipe.name}
+              width={150}
+              height={150}
+            />
+            <StyledFigcaption>{recipe.name}</StyledFigcaption>
+          </StyledFigure>
+        </li>
+      ))}
+    </RecipeStyledUl>
+  );
+}
+
+const StyledFigure = styled.figure`
+  position: relative;
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+  z-index: -1;
+  margin: 0;
+`;
+
+const StyledFigcaption = styled.figcaption`
+  font-size: 10px;
+  position: absolute;
+  left: 5%;
+  bottom: 10%;
+  color: black;
+  z-index: 1;
+`;
```

Comment:
- I prefer to have a wider range of z-index to helps prevent accidental overlaps or layout issues. 
- Using rem units for font-size instead of px can make your font size more accessible and responsive. 
- Specifying color using hexadecimal values is often preferred for consistency, especially in larger projects.

```suggestion
const StyledFigcaption = styled.figcaption`
  font-size: 0.8rem;
  position: absolute;
  left: 5%;
  bottom: 10%;
  color: #000;
  z-index: 10;
`;
```